src/Microsoft.Azure.EventHubs.Processor/Options/JsonConvertOptions.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System.Collections.Generic;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Serialization;
+
+namespace Microsoft.Azure.EventHubs.Processor.Options
+{
+    /// <summary>
+    /// JsonConvert Options
+    /// </summary>
+    public static class JsonConvertOptions
```

Comment:
>JsonConvertOptions [](start = 24, length = 18)

Instead of this, can you create a static instance of JsonSerializer? 

JsonSerializer.Create Method 
Creates a new JsonSerializer instance. The JsonSerializer will not use default settings from DefaultSettings.

[https://www.newtonsoft.com/json/help/html/M_Newtonsoft_Json_JsonSerializer_Create.htm](https://www.newtonsoft.com/json/help/html/M_Newtonsoft_Json_JsonSerializer_Create.htm)

Reply:
Yep, though you're creating a custom JsonSerializerSettings obj to be able to use the JsonConvert.SerializeObject function. 

I just checked another Azure [library](https://github.com/Azure/azure-webjobs-sdk/blob/663a508e8a851629c26a51e7de3af36629dfd120/src/Microsoft.Azure.WebJobs.Protocols/JsonSerialization.cs), they wrapped the entire json settings into a class, resulting pretty much the same I think...

We cannot create a non-default variant of the JsonConvert unfortunately. We could go for the webjob's implementation although we have to add the ContractResolver property otherwise I'm still conflicting with the snake case my code uses. (because they use JsonSerializer.CreateDefault it reuses the default AND the custom settings of the lib) Looking forward to your opinion on this :)